libopenage/engine/engine.cpp

```diff
@@ -53,31 +54,40 @@ Engine::Engine(mode mode,
 		this->time_loop.reset();
 	});
 
-	// if presenter is used, run it in a separate thread
+	log::log(INFO << "Using " << this->threads.size() + 1 << " threads "
+	              << "(" << std::jthread::hardware_concurrency() << " available)");
```

Comment:
The number is now no longer correct because simulation/presenter are started later. That's why I made the recommendation to start all threads in `loop()` in the other comment.